### PR TITLE
Feature/skale 1284 node backup procedure

### PR DESCRIPTION
**Pull request by dmytrotkk**

### CODE REVIEW

**Review by cstrangedk — APPROVED** [review]
lgtm. Also FYI @ebruoguzberk

File: core/node.py [hunk 1]
```diff
@@ -94,6 +97,24 @@ def init(env_filepath, dry_run=False):
     # todo: check execution result
 
 
+def init_backup(backup_path, env_filepath):
```
Comment (badrogger) [inline core/node.py]: I think we should rename it to restore

File: core/node.py [hunk 2]
```diff
@@ -133,3 +154,27 @@ def get_node_signature(validator_id):
...
+    time = datetime.datetime.utcnow().strftime("%Y-%m-%d-%H:%M:%S")
+    return f'{BACKUP_ARCHIVE_NAME}-{time}.tar.gz'
+
+
+def get_backup_filepath(base_path):
+    return os.path.join(base_path, get_backup_filename())
+
+
+def create_backup_archive(backup_filepath):
+    cmd = shlex.split(f'tar -zcvf {backup_filepath} -C {HOME_DIR} .skale')
```
Comment (badrogger) [inline core/node.py:176]: May be it's better to use tarfile module ?

File: core/node.py [hunk 3]
```diff
@@ -94,6 +97,24 @@ def init(env_filepath, dry_run=False):
...
+    res = subprocess.run(['bash', BACKUP_INSTALL_SCRIPT], env={
+        'SKALE_DIR': SKALE_DIR,
+        'DATAFILES_FOLDER': DATAFILES_FOLDER,
+        'BACKUP_RUN': 'True',
+        'BACKUP_PATH': backup_path,
+        'HOME_DIR': HOME_DIR,
+        **env_params
+    })
+    logger.info(f'Node init from backup script result: {res.stderr}, {res.stdout}')
+    print('Node restored from backup')
```
Comment (badrogger) [inline core/node.py]: If there're an error user still will receive "Node restored from backup"

**Review by badrogger — CHANGES_REQUESTED** [review]

File: core/node.py (hunk 2 above)
Comment (dmytrotkk) [reply core/node.py:176]: there were some problems in skale-admin with tarfile module (in `logs dump` cmd), so I decided to use system `tar` package here too

Comment (badrogger) [reply core/node.py:176]: ok, let's leave as is for now

File: core/node.py (hunk 3 above)
Comment (dmytrotkk) [reply core/node.py]: Addressed ✅

File: core/node.py (hunk 1 above)
Comment (dmytrotkk) [reply core/node.py]: Addressed ✅

**Review by badrogger — APPROVED** [review]
Great!

**Pull request merged**